docs/BindingsGuide_PowerShell.md

```diff
@@ -0,0 +1,206 @@
+# Azure SQL bindings for Azure Functions - PowerShell
+
+## Input Binding
+
+### function.json Properties for Input Bindings
+```json
```

Comment:
The intention I had for this section was to make it a table of properties and what they mean like this : https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql-input?tabs=in-process&pivots=programming-language-javascript#configuration

You can do it now if you want since that should be pretty straightforward (just copy the table since it should be the same content for the most part) but we could also save it and do them for all the languages at the same time so we're sure they have the same content.